README.md

```diff
@@ -55,24 +53,6 @@ The [**MSSQL Extension for Visual Studio Code**](https://www.aka.ms/vscode-mssql
   - Interact with each step in the execution plan, including collapsing or expanding nodes for a simplified view.
   - Zoom in or out to adjust the level of detail, or use "zoom to fit" for a complete view of the plan.
   - Highlight key performance indicators, such as elapsed time or subtree cost, to identify bottlenecks in query execution.
-- **Customizable Extension Options**: Configure command shortcuts, appearance, and other settings to personalize your development experience.
-
-## Public Preview Features
-
-- **GitHub Copilot Integration (`Preview`)**: Boost your productivity with AI-assisted SQL development
-  - Get intelligent code completions and suggestions for T-SQL as you type
-  - Build complex queries using natural language prompts
-  - Explain stored procedures, views, and joins to speed up onboarding and debugging
-  - Generate schema objects or test data in seconds
-  - Identify risky patterns and improve SQL security practices
-- **GitHub Copilot Agent Mode (`Preview`)**
-  - Use chat in Agent mode to connect, disconnect, or switch databases
-  - List servers and view available connections
-  - List all databases, tables, views, functions, and schemas in a connected server
-  - Run SQL queries directly from chat
-  - Get connection details for the current session
-  - Show database schema contextually
-  - Access all available Agent tools from the Agent Tools panel
 - **Local SQL Server Containers (`Preview`)**
```

Comment:
The `Preview` tag also needs be removed from the Local Container, Schema Compare, and Schema Designer sections that were moved to the GA header.

Reply:
I noticed the same while working on the last changes. Somehow, this change was not also included on my original change.

Fixed now.